GettingStarted/GettingStarted.md

```diff
@@ -1,40 +1,34 @@
-# Getting Started on the LSST Science Platform
+# Getting Started on the Rubin Science Platform
 
 _Greg Madejski and [Phil Marshall](https://github.com/LSSTScienceCollaborations/StackClub/issues/new?body=@drphilmarshall)_
 
 We are developing tutorial notebooks on remote JupyterLab instances, to short-circuit the DM stack installation process and get used to working in the
-notebook aspect of the LSST science platform. In these notes we provide:
-* [Notes on how to get set up on the LSST Science Platform (LSP) JupyterLab Notebook Aspect at the LSST Data Facility at NCSA](https://github.com/LSSTScienceCollaborations/StackClub/blob/master/GettingStarted/GettingStarted.md#accessing-the-lsst-science-platform)
+notebook aspect of the Rubin Science Platform (RSP). In these notes we provide:
+* [Notes on how to get set up on the Rubin Science Platform (RSP) JupyterLab Notebook Aspect at the LSST Data Facility at NCSA](https://github.com/LSSTScienceCollaborations/StackClub/blob/master/GettingStarted/GettingStarted.md#accessing-the-lsst-science-platform)
 * [Help with getting set up to run and edit the Stack Club tutorial notebooks](https://github.com/LSSTScienceCollaborations/StackClub/blob/master/GettingStarted/GettingStarted.md#running-and-contributing-to-the-stack-club-notebooks)
 
+## Accessing the Rubin Science Platform
+The [Rubin Science Platform (RSP) Notebook Aspect Documentation](https://nb.lsst.io/) provides an introduction to the system, including how to gain access and then how to use JupyterLab once you are in. 
+Accessing the RSP involves getting an NCSA account and figuring out VPN access.
```

Comment:
I think we need to add a statement like "RSP access is for individuals with data rights as described in ls.st/rdo-013" here.